Fixtures/Miscellaneous/EchoExecutable/Tests/TestSuite/Tests.swift

```diff
@@ -0,0 +1,7 @@
+import XCTest
+
```

Comment:
thought (non-blocking): Consider converting the XCTest to Swift Testing

A Canary Swift Testing test was introduced in #8222.  However, Converting some XCTest to Swift Testing failed in the CI pipelines (#8092, #8093, #8100) - which I did not get an opportunity to investigate.  However, this test is simple and may not encounter those errors.